lib/shopify_api/resources/vault_session.rb

```diff
@@ -0,0 +1,28 @@
+module ShopifyAPI
+  module VaultSessionFormat
```

Comment:
This is to avoid the patch here: https://github.com/Shopify/shopify_api/blob/master/lib/shopify_api/json_format.rb#L8.